whisper/cli/src/main.rs

```diff
@@ -215,8 +221,20 @@ fn execute<S, I>(command: I) -> Result<(), Error> where I: IntoIterator<Item=S>,
 	// Whisper protocol network handler
 	let whisper_network_handler = Arc::new(whisper::net::Network::new(pool_size, manager.clone()));
 
+	let network_config = {
+		let mut cfg = net::NetworkConfiguration::new();
+		let port = match &args.flag_port[..] {
+			"random" => 0 as u16,
+			port => port.parse::<u16>().expect("Invalid port specifier")
+		};
+		let addr = Ipv4Addr::from_str(&args.flag_address[..]).expect("Could not decode IP address");
```

Comment:
This will panic if an invalid IP Address is entered and the tests will not pass. Use to following instead:

```suggestion
		let addr = Ipv4Addr::from_str(&args.flag_address[..])?;
```